docs/sourcemaps.asciidoc

```diff
@@ -77,10 +77,10 @@ Send an example source map to the APM Server:
 ["source","sh",subs="attributes"]
 ---------------------------------------------------------------------------
 curl -X POST http://127.0.0.1:8200/v1/client-side/sourcemaps \
-  -F service_name="test-service" 
-  -F service_version="1.0" 
-  -F bundle_filepath="/static/js/main.js.map" 
-  -F sourcemap=@docs/data/intake-api/generated/sourcemap/bundle.js.map 
+  -F service_name="test-service" \
+  -F service_version="1.0" \
+  -F bundle_filepath="http://localhost/static/js/bundle.js" \
+  -F sourcemap=@bundle.js.map
```

Comment:
Why are you changing the location of the file? This is the path assuming you are in the root folder of the `apm-server`.

Reply:
@simitt , IMO, an example should be as simple as possible without much distraction, Since this example is supposed be read by devs who want to use that in their application, I don't think having a long path that is specific to the apm-server is good.

Reply:
I revisited my opinion on this, as I realized that most users probably have downloaded the apm-server package and not checked out the source code.